### DISCUSSION TITLE
Zoom plugin isn't working if Zoom and Boxselect plugins are implemented simultaneously in our Charts

### DISCUSSION THREAD

**Discussion by sitaramaraju555** [chartjs/chartjs-plugin-zoom#637] [Q&A]
I have imported CDNs in the following order to implement zoom and boxselect in my chartjs application:
-moment.js
-hammer.js
-zoom plugin
-bundle.js
-boxselect plugin

situation1 :
If we import all of them as shown above, the zoom functionality isn't working (No errors either)
situation2 :
If we comment out bundle.js, zoom works, but not boxselect

Any leads to get both functionalities working together?

**Comment by kurkle** [comment]
Converted to discussion as there is nothing to fix in this (without more information)
